src/marketing/prex-type-boughtbought.md

```diff
@@ -0,0 +1,22 @@
+---
+ee_only: true
+title: Bought this, bought that
+---
+
+The _Bought this, bought that_ recommendation type recommends products that tend to be purchased disproportionately more than others when purchasing the current item.
+
+## How it works
+
+The _Bought this, bought that_ recommendation type is most often on the cart or product detail page to increase exposure of related cross-sell products with the goal of increasing average order value. By aggregating what other shoppers have bought along with the current product, _Bought this, bought that_ displays highly relevant products shoppers can add to their cart.
+
+### Where used
+
+-  Product detail
+-  Cart
+-  Confirmation
+
+### Suggested storefront labels
+
+-  "Get everything you need"
```

Comment:
Do the labels contain the quotes? We are trying to be careful using them because it is often confusing that way.

Reply:
I removed them.